app/controllers/items_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Merchant::ItemsController < ApplicationController
+  before_action :set_merchant
+  before_action :set_item, only: [:show, :edit, :update, :destroy]
```

Comment:
also curious as to what this does!

Reply:
to be honest, I don't even know if I really need these.  It was something I found while researching after getting some errors.  But basically, they're filters that define common behavior for certain actions.  They're not necessary though; I'm going to take them out and repush.